doc/Language/control.pod6

```diff
@@ -39,24 +39,38 @@ executed.
     { say 2; say 3 };         # OUTPUT: «2␤3␤»
     say 4;                    # OUTPUT: «4␤»
 
-Unless it stands alone as a statement, a block simply creates a closure. The
-statements inside are not executed immediately. Closures are another topic
-and how they are used is explained
-L<elsewhere|/language/functions#Blocks_and_lambdas>. For now it is just
-important to understand when blocks run and when they do not:
+This might be useful if you are using an editor that offers code folding,
+so you can use these blocks to add visual sections to your code. 
```

Comment:
Well, this describes intent, only you can simply omit the {} and it will still do the same and look pretty much the same. The example was probably flawed to start with. Blocks are literals for functions, and they should be used as such. They are code.